concepticondata/concepticon.tsv

```diff
@@ -3060,3 +3060,58 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
+3093	BRUSH (TEETH)	Clothing and grooming	Clean one's teeth by repeatedly grating over them with an object.	Action/Process	
+3094	CLAP	Motion	Make noise by putting one's hand together.	Action/Process	
+3095	CONDUCT	Speech and language	To lead some kind of musical performance.	Action/Process	
+3096	ERUPT	The physical world	Burst of matter from a volcano.	Action/Process	
+3097	FILE	Clothing and grooming	Make something smooth, often nails or other parts of the body.	Action/Process	
+3098	GRATE	Basic actions and technology	To abrade an object by rubbing it against a rough surface.	Action/Process	
+3099	HAMMER (SOMETHING)	Basic actions and technology	Strike or beat something with a hammer.	Action/Process	
+3100	HATCH	Animals	Emergence from an egg.	Action/Process	
+3101	HITCHHIKE	Motion	Travel by means of lifts in vehicles.	Action/Process	
+3102	HUG	Social and political relations	The act of embracing someone or something with one's arms.	Action/Process	
+3103	IRON (CLOTHES)	Clothing and grooming	To make clothes smooth by brushing over them with a heated tool.	Action/Process	
+3104	MASSAGE	Basic actions and technology	To rub someone's body with the hands.	Action/Process	
+3105	MOP	Basic actions and technology	To clean or wipe up.	Action/Process	
+3106	OPERATE (MEDICINE)	The body	Perform a medical procedure by cutting or making incisions.	Action/Process	
+3107	PAINT (SOMETHING)	Basic actions and technology	The act of spreading coloured pigments on a surface.	Action/Process	
+3108	URINATE	The body	Excrete urine from the body.	Action/Process	
+3109	PHOTOGRAPH (SOMETHING)	Modern world	Use a light-sensitive device to take an image of something.	Action/Process	
+3110	PLAY (INSTRUMENT)	Speech and language	Execute coordinated movements on or with a musical instrument to produce sounds.	Action/Process	
+3111	PLAY (GAME)	Social and political relations	Act within the frame and rules of a game.	Action/Process	
+3112	POST (SOMETHING)	Basic actions and technology	To send something by means of a postal service.	Action/Process	
+3113	RAKE (SOMETHING)	Basic actions and technology	The act of moving a tool over a surface in order to level or clean it.	Action/Process	
+3114	SHOWER	Clothing and grooming	Douse something in a spray or stream of water.	Action/Process	
+3115	SKATE	Motion	Move swiftly over a smooth or slippery surface.	Action/Process	
+3116	SPILL	Motion	Cause liquids to flow over the edge or out of a container.	Action/Process	
+3117	SUNBATHE	The body	Sit or lie in the sun in order to tan the skin.	Action/Process	
+3118	SURF	Motion	Ride on waves by using a flat object.	Action/Process	
+3119	TALK (PHONE)	Speech and language	To communicate by means of a telephone.	Action/Process	
+3120	TYPE (ON SOMETHING)	Modern world	Using keys of a computer or typewriter.	Action/Process	
+3121	VACUUM	Modern world	Clean something by using a vacuum cleaner.	Action/Process	
```

Comment:
VACUUM (SOMETHING), again, this would be in danger of invoking a nominal reading.

Reply:
Agree.